### PR TITLE
Add a long list of missing params and properties

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @brandur-stripe 
cc @mickjermsurawong-stripe who found all of those for us.
cc @stripe/api-libraries

### CODE REVIEW

File: token.go [hunk 1]
```diff
@@ -46,6 +46,6 @@ type Token struct {
 
 // PIIParams are parameters for personal identifiable information (PII).
 type PIIParams struct {
-	Params           `form:"*"`
-	PersonalIDNumber *string `form:"personal_id_number"`
+	Params   `form:"*"`
+	IDNumber *string `form:"id_number"`
```
Comment (remi-stripe) [inline token.go]: This is a breaking change and likely requires a major version to play it safe even if it likely did not work.

File: refund.go [hunk 2]
```diff
@@ -38,6 +38,7 @@ type RefundParams struct {
 // For more details see https://stripe.com/docs/api#list_refunds.
 type RefundListParams struct {
 	ListParams `form:"*"`
+	Charge     *string `form:"charge"`
```
Comment (mickjermsurawong-stripe) [inline refund.go]: One more created range here pls Remi
https://stripe.com/docs/api/refunds/list#list_refunds-created

**Comment by remi-stripe** [comment]
@mickjermsurawong-stripe IMO we should not keep such a PR open for a week. There's no real upside to "waiting until we find more", we would just do a brand new PR at that point. 

I also think the empty params should be resolved entirely separately and not in this PR. We should also talk about the best approach internally instead.

**Comment by mickjermsurawong-stripe** [comment]
ah that's fair. I'm good with merging this then
LGTM

**Comment by brandur-stripe** [comment]
> I also think the empty params should be resolved entirely separately and not in this PR. We should also talk about the best approach internally instead.

+1. I think we're going to need to add a new property (e.g. `Empty`) and custom encode function to support this properly unfortunately.

**Comment by mickjermsurawong-stripe** [comment]
The build was stuck but seems working now. I can take the release

**Comment by remi-stripe** [comment]
@brandur-stripe yeah I canceled and restarted. Leaving the release to you, thanks a lot! Reminder that it's a major version to be safe

**Comment by brandur-stripe** [comment]
Released as 60.0.0.

**Comment by mickjermsurawong-stripe** [comment]
ah thank you @brandur! i'm just about reading instructions and preparing the release message!

**Comment by brandur-stripe** [comment]
> ah thank you @brandur-stripe! i'm just about reading instructions and preparing the release message!

Oops, sorry @mickjermsurawong-stripe — I didn't mean to step on your toes. Somehow I read the "you" in as meaning me, hah (which reading through again makes no sense I see).

**Pull request merged**